frontend/src/views/coverage/enrollment/ResidentDetails.vue

```diff
@@ -0,0 +1,385 @@
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col7">
+          <AppInput :e-model="v$.address2" id="address2" label="Line 2 (Optional)" v-model="address2" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col7">
+          <AppInput :e-model="v$.address3" id="address3" label="Line 3 (Optional)" v-model="address3" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col4">
+          <AppInput :e-model="v$.city" id="city" label="City" type="text" v-model.trim="city" />
+        </AppCol>
+        <AppCol class="col4">
+          <AppSelect :e-model="v$.province" id="province" label="Province" v-model="province" :options="provinceOptions" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col4">
+          <AppInput :e-model="v$.postalCode" id="postalCode" label="Postal Code" type="text" v-model.trim="postalCode" />
+        </AppCol>
+      </AppRow>
+
+      <AppRow>
+        <AppCol class="col7">
+          <AppInput :e-model="v$.mailingAddress1" id="mailingAddress1" label="Mailing Address (if different from home address)" v-model="mailingAddress1" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col7">
+          <AppInput :e-model="v$.mailingAddress2" id="mailingAddress2" label="Line 2 (Optional)" v-model="mailingAddress2" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col7">
+          <AppInput :e-model="v$.mailingAddress3" id="mailingAddress3" label="Line 3 (Optional)" v-model="mailingAddress3" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col4">
+          <AppInput :e-model="v$.mailingAddressCity" id="mailingAddressCity" label="City" type="text" v-model.trim="mailingAddressCity" />
+        </AppCol>
+        <AppCol class="col4">
+          <AppSelect :e-model="v$.mailingAddressProvince" id="mailingAddressProvince" label="Province" v-model="mailingAddressProvince" :options="provinceOptions" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col4">
+          <AppInput :e-model="v$.mailingAddressPostalCode" id="mailingAddressPostalCode" label="Postal Code" type="text" v-model.trim="mailingAddressPostalCode" />
+        </AppCol>
+      </AppRow>
+
+      <AppRow>
+        <AppCol class="col4">
+          <AppSelect :e-model="v$.priorResidenceCode" id="priorResidenceCode" label="Prior Residence Code" v-model="priorResidenceCode" :options="priorResidenceOptions" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col4">
+          <AppInput
+            :e-model="v$.otherProvinceHealthcareNumber"
+            id="otherProvinceHealthcareNumber"
+            label="Other Province Healthcare Number (If Applicable) (Optional)"
+            type="text"
+            v-model.trim="otherProvinceHealthcareNumber"
+          />
```

Comment:
Formatting here is wrapped. Others aren't.

Reply:
This is because this line is longer, formatting is set to wrap lines over 200 in width as I find anything longer is not reader friendly. Might need to standardize this.